### PR TITLE
Feature/content changes5

### PR DESCRIPTION
**Pull request by richardverheyen**
I'm actually really proud of my work on this today, I've fixed everything I set out to fix with css and html. The only thing I still would like to learn is how to use .js to make elements animate in when you scroll up to them (especially those on fold 2 of landing, and the services page elements). I found some examples of .js on the internet, but I wasnt sure if I should just start tacking on other people's code to the main.js file without asking, so I decided to wait - because it seems like it will be relatively straightforward for you to fix, or teach me. 

The major fixes in this PR are the #hero down arrow, article page back buttons, and and #articles section reactivity. With a few minor arrangements and spell checks scattered throughout.

### CODE REVIEW

**Comment by richardverheyen** [comment]
Oh, there is an issue with the address bar when I link to an id element (link when I go back from an article post, or I link through from the unique button on the landing page) and the address bar displays ~/legislation-changes/#articles

**Comment by janwerkhoven** [comment]
Sweet :)

Is this WIP or is it ready for review?

What about the other PR you said I should ignore? Best to close that PR.

Quiet common is for us to name the PR "[WIP] Some sensible name" until it's ready for review. Once ready you can just remove "[WIP] " from the name.

**Comment by janwerkhoven** [comment]
Ow and you might want to do `git pull origin staging` :)

File: src/public/assets/img/arrow-left-hover.svg [hunk 1]
```diff
@@ -4,11 +4,11 @@
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 283.5 283.5" enable-background="new 0 0 283.5 283.5" xml:space="preserve">
 <g>
-	<path fill="#0D344C" d="M141.7,0c78.5,0,141.7,63.2,141.7,141.7s-63.2,141.7-141.7,141.7S0,220.2,0,141.7S63.2,0,141.7,0z
+	<path fill="$finsec-royal-blue" d="M141.7,0c78.5,0,141.7,63.2,141.7,141.7s-63.2,141.7-141.7,141.7S0,220.2,0,141.7S63.2,0,141.7,0z
```
Comment (janwerkhoven) [inline src/public/assets/img/arrow-left-hover.svg:7]: 😱  Good job

File: src/styles/_common.scss [hunk 2]
```diff
@@ -236,6 +236,9 @@ i {
     height: 160vw;
     @include viewport-unit(height, 160vw);
   }
+  @media (max-width: 400px) {
+    height: 100vh;
```
Comment (janwerkhoven) [inline src/styles/_common.scss:240]: Nooo! Don't. This cause mobile screens to jerk each time the URL bar hides and shows.

Comment (janwerkhoven) [inline src/styles/_common.scss:240]: I'll merge it in and fix this afterwards.

**Comment by janwerkhoven** [comment]
⛵

**Pull request merged**